data/presets/amenity/taxi/auto_rickshaw.json

```
@@ -0,0 +1,31 @@
+{
+    "icon": "fas-person-arrow-up-from-line",
+    "fields": [
+        "name",
+        "address"
+    ],
+    "moreFields": [
+        "access_simple",
+        "opening_hours",
+        "operator",
+        "wheelchair"
+    ],
+    "geometry": [
+        "point",
+        "area"
+    ],
+    "terms": [
```

add `bajaj` as a search term?

```suggestion
    "terms": [
        "bajaj",
```